scripts/post_deploy_sns.py

```diff
@@ -38,16 +38,15 @@ def sns_topic_arn() -> str:
     """
     Return the ARN of the SNS topic.
     """
-    return ':'.join(
-        (
-            'arn',
-            'aws',
-            'sns',
-            config.region,
-            config.aws_account_id,
-            config.sns_topic_qualified_name
-        )
+    parts = (
```

Comment:
Please address my previous comment on this.

Reply:
It appears I am unclear on the requested change here. You referenced the "Collection literals" section of the contribution guide, which suggests tuples should be used in the case of heterogeneous elements and of fixed length, and lists should be used in the case of homogeneous elements of varying length.

In the situation here we have a homogeneous collection of fixed length. With one criteria from each of the two collection types, I opted for a tuple due to the nature of this collection only existing to be joined into a string. The "String concatenation" section of the contributing guide indirectly implies this is okay with the example `','.join((a, b, c, d))`.

From your comment I interpret that I came to the wrong conclusion.
Searching the codebase I find:
```
join([
    ...
])
```
more common than
```
join((
    ...
))
```
so I have added a fixup to match convention.